### PR TITLE
[develop]es6 changes to files under lib, examples and test

### PR DESCRIPTION
**Pull request by manoj9788**
In continuation to my previous PR
@grawk

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,23 +214,17 @@ You'll see the `plugins.cookie` section, which is loading `examples/plugin/nemo-
 'use strict';
 
 module.exports = {
-  "setup": function (nemo, callback) {
+  "setup"(nemo, callback) {
```
Comment (skratchdot) [inline README.md]: missing colon. it should be:

`"setup": (nemo, callback) {`

File: examples/plugin/nemo-cookie.js [hunk 2]
```diff
@@ -1,29 +1,17 @@
 'use strict';
 
 module.exports = {
-  "setup": function (nemo, callback) {
+  "setup"(nemo, callback) {
```
Comment (skratchdot) [inline examples/plugin/nemo-cookie.js]: missing colon

File: lib/configure.js [hunk 3]
```diff
@@ -69,13 +69,13 @@ module.exports = function Configure(_basedir, _configOverride) {
...
-  var returnJSON = {};
-  var originalValue = process.env[prop];
+let envToJSON = prop => {
+  const returnJSON = {};
+  const originalValue = process.env[prop];
   if (originalValue === undefined) {
     return {
       'json': {},
-      'reset': function () {
+      'reset'() {
```
Comment (skratchdot) [inline lib/configure.js]: missing colon

File: lib/configure.js [hunk 4]
```diff
@@ -88,7 +88,7 @@ var envToJSON = function (prop) {
   }
   return {
     'json': returnJSON,
-    'reset': function () {
+    'reset'() {
```
Comment (skratchdot) [inline lib/configure.js]: missing colon

File: README.md (hunk 1 above)
Comment (manoj9788) [inline README.md]: @skratchdot Ah! so only the export default doesn't need the use of colon? - I wondering how the tests passed and didnt show any lint errors

Comment (skratchdot) [inline README.md]: Okay. Disregard these comments.  I guess it was the "quotes" around the function name that was throwing me off.  I guess these changes are okay.

https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=env&targets=&browsers=&builtIns=false&debug=false&code='use strict'%3B%0A%0Amodule.exports %3D {%0A setupa(nemo%2C callback) {}%2C%0A "setupb"(nemo%2C callback) {}%2C%0A setupc%3A (nemo%2C callback) %3D> {}%0A}%3B

Comment (manoj9788) [inline README.md]: cool :)

Comment (manoj9788) [inline README.md]: I will now fix the Travis build errors.

**Comment by manoj9788** [comment]
any update here ?

**Comment by grawk** [comment]
Very sorry for the delay... Looks good to me.

**Comment by grawk** [comment]
One small thing. Would you mind adding a CHANGELOG entry please @manoj9788 ?

**Comment by nikulkarni** [comment]
This looks good to me, and open for long time. Can we merge this in and create CHANGELOG entry as a follow up PR?

**Comment by skratchdot** [comment]
It looks good to me. IMO, we could update the changelog when we bump the version again.

**Comment by nikulkarni** [comment]
Alright merging it

**Comment by manoj9788** [comment]
Was away for past few days! Thanks for getting this merged @nikulkarni

**Pull request merged**